build.sh

```diff
@@ -204,18 +228,25 @@ cmake_boolean_option()
 	esac
 }
 
+# ---------------------------------------------------------------------------
+# Let's process the user's wishes
+# ---------------------------------------------------------------------------
+
+features_set_to_autodetect
+parse_args "$@"
+
+if [ $PRINT_HELP -ne 0 ] ; then
+	print_help
+	exit 1
+fi
+
+MAKE_TASKS=$(num_cpu)
```

Comment:
This assignment overrides the value set by parse_args, causing -j to be ignored.